ServerHttp/src/main/java/io/raspberrywallet/contract/IncorrectPasswordException.java

```diff
@@ -0,0 +1,7 @@
+package io.raspberrywallet.contract;
+
+public class IncorrectPasswordException extends Exception {
```

Comment:
other exceptions in contract are extending Throwable class (not sure why), shouldn't we follow this?